### ISSUE TITLE
The robot is not moving forward

### ISSUE THREAD

**Issue by deardost** [chvmp/champ#46]
Hi there
I tried launching the files below:
![Screenshot from 2021-03-23 19-28-03](https://user-images.githubusercontent.com/74898176/112158710-6f51e180-8c0e-11eb-87ab-1fd5f54a05db.png)

But I ran into the following problems:
1. Gazebo environment is running in a very slow speed. (My system is Core i5, with 8 gb ram and 2 gb AMD Radeon Graphics Card. I am using Ubuntu 20.04.2 LTS and ROS Noetic. It's installed in SSD). I wonder if my system specs are not sufficient for normal speed.
2. The robot is doing somewhat rotation almost around it's axis. Would you please tell me why it happens and what's the solution?
Thanks in advance for all this help
I have attached a video below to show the warnings that I get and to show what the robot does:

https://user-images.githubusercontent.com/74898176/112160318-0c614a00-8c10-11eb-8307-024c079ac467.mp4

**Comment by grassjelly** [comment]
Your RTF's way too low. Your problem might be related to this: https://github.com/chvmp/champ/issues/14

Alternatively if you just want to try walking the robot, you can try simulating without the assets:

```
roscd champ_config/worlds
roslaunch champ_config gazebo.launch gazebo_world:=default.world
```

and use teleop to control the robot

**Comment by peci1** [comment]
Solution for proper running using simulation time has been just proposed: https://github.com/chvmp/champ/pull/48 .

**Comment by deardost** [comment]
@peci1 Forgive me, but I don't understand your proposed solution. I am a noob in github. What do I need to do with the package (champ) that I have cloned to make it run smoothly and not slow?

**Comment by deardost** [comment]
@grassjelly I want full navigation for our robot. We are working on a project. It's quadruped mobile robot. So we want our robot to navigate to the target location, using gps

**Comment by peci1** [comment]
Just call `git pull --recurse-submodules` in the directory where you cloned champ. I'm not sure about Noetic compatibility, but I don't see any general problem for it to not work.

**Comment by deardost** [comment]
@peci1 nope. That didn't solve my issue. Robot is still spinning around before it reaches final destination.
